src/frontend/komponenter/Fagsak/Simulering/SimuleringTabell.tsx

```diff
@@ -58,27 +62,57 @@ const SimuleringTabellOverskrift = styled.div`
     margin-bottom: 1rem;
 `;
 
+const ElementMedFarge = styled(Element)`
+    color: ${(props: { farge?: string }) => (props.farge ? props.farge : navFarger.navMorkGra)};
+`;
+
 interface ISimuleringProps {
     simulering: ISimuleringDTO;
 }
 
-const SimuleringTabell: React.FunctionComponent<ISimuleringProps> = ({
-    simulering: { fomDatoNestePeriode, fom, perioder, tomDatoNestePeriode },
-}) => {
+const SimuleringTabell: React.FunctionComponent<ISimuleringProps> = ({ simulering }) => {
+    const {
+        fomDatoNestePeriode,
+        fom,
+        perioder: perioderUtenTommeSimuleringer,
+        tomDatoNestePeriode,
+    } = simulering;
+    const { hentPerioderMedTommePerioder, hentÅrISimuleringen } = useSimulering();
+    const årISimuleringen = hentÅrISimuleringen(perioderUtenTommeSimuleringer);
+    const perioder = hentPerioderMedTommePerioder(perioderUtenTommeSimuleringer);
+    const [indexFramvistÅr, settIndexFramistÅr] = useState(årISimuleringen.length - 1);
+    const aktueltÅr = årISimuleringen[indexFramvistÅr];
+
     const kapitaliserTekst = (tekst: string): string => {
         return tekst.charAt(0).toUpperCase() + tekst.slice(1).toLowerCase();
     };
 
-    const formaterBeløpUtenPostfiks = (beløp: number) => formaterBeløp(beløp).slice(0, -3);
+    const periodeErEtterNesteUtbetalingsPeriode = (periode: ISimuleringPeriode) =>
+        fomDatoNestePeriode && dayjs(periode.fom).isAfter(dayjs(fomDatoNestePeriode));
 
     const periodeSkalVisesITabell = (periode: ISimuleringPeriode) =>
-        !fomDatoNestePeriode || !dayjs(periode.fom).isAfter(dayjs(fomDatoNestePeriode));
+        !periodeErEtterNesteUtbetalingsPeriode(periode) && dayjs(periode.fom).year() === aktueltÅr;
+
+    const formaterBeløpUtenPostfiks = (beløp?: number) =>
```

Comment:
Denne var litt magisk, tror kanskje vi har utilfunksjon for formatering av beløp også.